telebot/asyncio_helper.py

```diff
@@ -59,7 +59,10 @@ async def get_session(self):
 async def _process_request(token, url, method='get', params=None, files=None, request_timeout=None):
     params = _prepare_data(params, files)
     if request_timeout is None:
-        request_timeout = REQUEST_TIMEOUT
+        if params and ("timeout" in params):
```

Comment:
As you can see [here](https://github.com/eternnoir/pyTelegramBotAPI/blob/master/telebot/async_telebot.py#L146), get_updates has 2 params: request_timeout & timeout. Timeout should be passed to telegram, meanwhile request_timeout is supposed to go to aiohttp. So, if the method is not getUpdates, then it should pop the timeout from dictionary. For getUpdates, request_timeout and timeout should be passed without changes.

Reply:
> So, if the method is not getUpdates, then it should pop the timeout from dictionary

@coder2020official Please, check.